zip-codes/scripts/search.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
 (function(module) {
-
+  webDB.execute(
+    'SELECT DISTINCT state FROM zips ' +
+    'ORDER BY state DESC;',
+    function(rows){
+      console.log(rows);
+      if (rows.length){
+        rows.forEach(function(a){
+          $('#state-select').append('<option>' + a.state + '</option>');
+        });
+      }
+      $('#state-select').on('change', function() {
+        $('#city-select').find('option').remove().end();
```

Comment:
Great work emptying out the select box before appending cities! Alternatively, and perhaps more concise: `$('#city-select').empty()`